tests/unit/s2n_config_test.c

```diff
@@ -74,10 +78,14 @@ int main(int argc, char **argv)
         {
             EXPECT_SUCCESS(s2n_enable_tls13());
             struct s2n_connection *conn;
+            const struct s2n_security_policy *security_policy;
             EXPECT_NOT_NULL(conn = s2n_connection_new(S2N_CLIENT));
 
             EXPECT_EQUAL(conn->config, s2n_fetch_default_config());
-            EXPECT_EQUAL(conn->config->security_policy, tls13_security_policy);
+
+            EXPECT_SUCCESS(s2n_connection_get_security_policy(conn, &security_policy));
+            EXPECT_NOT_NULL(security_policy);
+            EXPECT_EQUAL(security_policy, tls13_security_policy);
```

Comment:
Nit: Are these expect_not_nulls necessary, since you do an equals check immediately after?

Reply:
Its not necessary, I can get rid of it.